dora/tests/integration/src/test/java/alluxio/client/fs/io/FileInStreamIntegrationTest.java

```diff
@@ -463,220 +411,9 @@ public void positionedReadWithoutCaching() throws Exception {
 
       FileInStream is = mFileSystem.openFile(uri, FileSystemTestUtils.toOpenFileOptions(op));
       byte[] ret = new byte[DELTA - 1];
-      Assert.assertEquals(DELTA - 1, is.positionedRead(MIN_LEN - DELTA + 1, ret, 0, DELTA));
+      Assert.assertEquals(DELTA - 1, is.positionedRead(MIN_LEN - DELTA + 1, ret, 0, DELTA - 1));
       Assert.assertTrue(BufferUtils.equalIncreasingByteArray(MIN_LEN - DELTA + 1, DELTA - 1, ret));
       is.close();
     }
   }
-
-  @Test
-  @LocalAlluxioClusterResource.Config(
-      confParams = {PropertyKey.Name.USER_FILE_SEQUENTIAL_PREAD_THRESHOLD, "700KB"})
```

Comment:
the property `pread.threashold` is deprecated, so related test is no longer needed